R/ResultFlagsDependent.R

```diff
@@ -412,17 +412,17 @@ InvalidResultUnit <- function(.data, clean = c("invalid_only", "nonstandardized_
   #                   .after = "ResultMeasure.MeasureUnitCode"
   #   )
   
-  # if all rows are "Valid", return input unchanged
+  # if all rows are "Valid", return input with flag column
   if (any(c("Nonstandardized", "Invalid") %in%
           unique(check.data$TADA.ResultUnit.Flag)) == FALSE) {
-    print("All data are valid, therefore the function cannot be applied.")
-    .data = OrderTADACols(.data)
-    return(.data)
+    print("All characteristic/unit combinations are valid in your dataframe. Returning input dataframe with TADA.ResultUnit.Flag column.")
```

Comment:
Suggestion: Returning input dataframe with added TADA.ResultUnit.Flag column for tracking.